src/test/java/com/google/devtools/build/lib/bazel/rules/android/AndroidSdkRepositoryTest.java

```diff
@@ -38,6 +38,7 @@ public void setup() throws Exception {
     scratch.file(
         "bazel_tools_workspace/tools/android/android_sdk_repository_template.bzl",
         ResourceLoader.readFromResources("tools/android/android_sdk_repository_template.bzl"));
+    scratch.appendFile("WORKSPACE", "local_config_platform(name='local_config_platform')");
```

Comment:
Why is this change needed?

Reply:
The androidsdk repo reads constraints.bzl from `@local_config_platform` (to determine the host constraints for the `exec_compatible_with`), so it needs to be defined.